### PR TITLE
Dropzone update

### PR DESCRIPTION
**Pull request by sihu**
Changes from 4.0.1 to version 4.3.0 due to https://github.com/enyo/dropzone/releases

### CODE REVIEW

File: dropzone/dropzone.d.ts [hunk 1]
```diff
@@ -45,10 +47,12 @@ interface DropzoneOptions {
 	clickable?: boolean|string|HTMLElement|(string|HTMLElement)[];
 	ignoreHiddenFiles?: boolean;
 	acceptedFiles?: string;
+	renameFilename?: RenameFilenameFunction;
```
Comment (outring) [inline dropzone/dropzone.d.ts]: It's fine, but to get more clear IDE suggestion and make it more readable is better to define it as a function, like this: `renameFilename?(name:string):string`

**Comment by outring** [comment]
:+1:

File: dropzone/dropzone.d.ts (hunk 1 above)
Comment (sihu) [inline dropzone/dropzone.d.ts]: thx for your review. i changed it like this!

**Comment by sihu** [comment]
@outring i improved the declaration. do you think, it's now ready to merge?

**Comment by outring** [comment]
@sihu, looks great! Now we have to wait for somebody who will merge it :D

**Comment by outring** [comment]
@vvakame :+1:

**Pull request merged**